newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -906,6 +916,30 @@ const declarePropertyInstructionAndExpression = (
       .addParameter('yesorno', i18n._(t`New value to set`), '', false)
       .getCodeExtraInformation()
       .setFunctionName(setterName);
+
+    addObjectAndBehaviorParameters(
+      entityMetadata.addScopedAction(
+        toggleActionName,
+        i18n._(t`Toggle ${propertyLabel}`),
+        i18n._(
+          t`Toggle the property value for ${'"' + uncapitalizedLabel + '"'}.`
```

Comment:
Sorry for not being clear, I was confused with the other sentence. Here we can have:
```suggestion
          t`Toggle the property value for "${uncapitalizedLabel}".`
```

(quotes are part of the translated string)